hypershift-operator/controllers/hostedcluster/internal/platform/agent/agent.go

```diff
@@ -10,12 +10,21 @@ import (
 	"github.com/openshift/hypershift/hypershift-operator/controllers/manifests/ignitionserver"
 	"github.com/openshift/hypershift/support/upsert"
 	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
 	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	k8sutilspointer "k8s.io/utils/pointer"
 	capiv1 "sigs.k8s.io/cluster-api/api/v1beta1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	//TODO set image from config?
+	imageCAPA = "quay.io/rollandf/bm-inventory:capiagent" //"quay.io/edge-infrastructure/cluster-api-provider-agent:latest-2021-12-30"
```

Comment:
CAPA is taken for AWS

Reply:
@enxebre that is correct, there is a build pipeline through Openshift.
@rollandf I think the namespace and multi instance support is already merged and you can use the latest image from the above pipeline `quay.io/edge-infrastructure/cluster-api-provider-agent:latest`

Reply:
Done